src/resonanceReconstruction/rmatrix/src/coh3-coulomb.hpp

```diff
@@ -1,9 +1,9 @@
-const double HBAR       =  6.582119514e-22; /* Planck's constant/2pi [MeV sec]*/
-const double VLIGHT     =  2.99792458e+23 ; /* light velocty [fm/sec]         */
-const double AMUNIT     =  931.4943335    ; /* MeV = 1amu                     */
-const double COULOMB    =  1.60217733e-19 ; /* J = 1eV                        */
-const double COULOMBSQ  =  2.56697220e-38 ; /* COULOMB*COULOMB                */
-const double PERMITTIV  =  5.60958617e+37 ; /* permittivity [MeV fm /C^2]     */
+//const double HBAR       =  6.582119514e-22; /* Planck's constant/2pi [MeV sec]*/
```

Comment:
When we have figured out why our tests are not quite correct, we should delete these lines instead of commenting them out.